032-MLOpsFromScratch/Student/03-BuildPipeline.md

```diff
@@ -0,0 +1,89 @@
+# Challenge 3 – Create a Build Pipeline in Azure DevOps
+
+[< Previous Challenge](./02-UnitTesting.md) - **[Home](../README.md)** - [Next Challenge >](./04-ReleasePipeline.md)
+
+## Introduction
+
+The Build pipeline demonstrates the automation of various stages/tasks involved
+in building an ML model and creating a container image on top of that model. The
+stages generally constitute creating a project environment, preparing the data,
+training the model, evaluating the model, registering/versioning the model, and
+scoring the model generally by containerizing the model.
+
+There are several ways to create a Build pipeline. The two most common and popular ways
+are:
+
+-   using a YAML file that represents the entire pipeline,
+
+-   using an empty job and adding tasks sequentially
```

Comment:
Capitalize "Using" in both bullets and use a period or no ending punctuation at all.

Reply:
Done